jms/1.7/modules/ROOT/pages/jms-topic-subscription.adoc

```diff
@@ -1,17 +1,29 @@
-= Using Topic Subscriptions
+= Configure Topic Subscriptions with JMS Connector
 :keywords: jms, connector, consume, source, listener, topic, subscription
 :page-aliases: connectors::jms/jms-topic-subscription.adoc
 
+Topics in JMS enables you to configure how the subscription behaves regarding the connection of the consumer and also how these consumers interact with each other.
 
+The Anypoint Connector for JMS (JMS Connector) *Consumer* and *On New Message* operations can manage the aspects of a topic subscription in the *Consumer type* field by selecting the *Topic consumer* configuration.
 
-Topics in JMS allow you to configure how the subscription behaves regarding the connection of the consumer and also how this consumers interact between each other.
+== Configure Durable Subscriptions
 
-Using Mule's JMS Connector you can handle of the aspects of a Topic subscription using the 'topic-consumer' configuration in both the `consumer` and `listener` operations.
+Durable subscriptions enables you to receive published messages to the topic while the consumer is offline.
+Once you configure a durable subscription, an available consumer receives all the published messages before the established connection. +
```

Comment:
GLOBALLY replace "once" with either "after" or "one time," depending on your intent (translation):
```suggestion
After you configure a durable subscription, an available consumer receives all the published messages before the established connection. +
```
Isn't this just repeating the previous sentence? Are we trying to clarify that durable subscriptions enable all subscribed consumers to receive one another's published messages, even without a current connection?